Makefile

```diff
@@ -1,5 +1,7 @@
 CXX=g++
 CCFLAGS=-Wall -Werror -O3 -g -std=c++17
 
-all:
-	$(CXX) $(CCFLAGS) test.cc -o test 
+test:
+	$(CXX) $(CCFLAGS) test.cc -o test
+benchmark:
+	$(CXX) $(CCFLAGS) benchmark.cc -o test
```

Comment:
Here is a quick Makefile I wrote to give you a sense of what a slightly more advanced one would look like.

```
#CXX = g++
CXX = clang++
CFLAGS = -Wall -Werror -Wextra -pedantic -O2 -g -std=c++17
DBG_CFLAGS = -Wall —g Werror -Wextra -pedantic -std=c++17

.PHONY: all clean

std:
	g++ test.cc -o test -DUSE_STD_MAP
chained:
	g++ test.cc -o test -DUSE_CHAINED_MAP

libobj = StdMapHashTable.o
exeobj = test1.o

all: test1

test1: $(libobj) $(exeobj)
	$(CXX) $(CFLAGS) -o test1 $(exeobj) $(libobj)

%.o: %.cc
	$(CXX) -c $(CFLAGS) -o $@ $<

%.go: %.cc
	$(CXX) -c $(DBG_CFLAGS) $*.cc -o $*.go

clean:
	rm -f test *.o *.go *.d
```